easybuild/easyblocks/f/flexiblas.py

```diff
@@ -104,20 +103,21 @@ def configure_step(self):
         if build_option('rpath'):
             self.cfg['abs_path_compilers'] = True
 
-            # inject RPATH link option for build location of libflexiblas.so.3,
-            # to ensure that test binaries can find the FlexiBLAS library
-            build_libdir = os.path.join(builddir, 'lib')
-            for envvar in ['CFLAGS', 'CXXFLAGS', 'FFLAGS']:
-                flags = os.getenv(envvar)
-                setvar(envvar, '-Wl,-rpath=%s %s' % (build_libdir, flags))
-
         super(EB_FlexiBLAS, self).configure_step(builddir=builddir)
 
     def test_step(self):
         """Run tests using each of the backends."""
 
+        additional_pretestopts = ''
+        if build_option('rpath'):
+            # inject build location of libflexiblas.so.3,
+            # to ensure that test binaries can find the FlexiBLAS library
+            build_libdir = os.path.join(self.builddir, 'easybuild_obj', 'lib')
+            additional_pretestopts = "LD_LIBRARY_PATH=%s:$LD_LIBRARY_PATH" % build_libdir
```

Comment:
`$LD_LIBRARY_PATH` is ignored when RPATH is involved, so this is pointless?

I'm working on a similar change though: only injecting `-Wl,-rpath=...` into `$FFLAGS` & co for the test command  (via `testopts`), testing that now...

Reply:
For POWER, may need to include `lib64`?